docs_outdoornav_user_manual/api/api_endpoints/mission_scheduler_api.mdx

```diff
@@ -0,0 +1,172 @@
+to write your own interface or create automation tools to interact with the scheduler.
+
+## The Schedule Object
+
+A schedule consists of one or more missions to be executed in the specified order at a particular start time. For
+consistency the start time is expressed as a 24-hour time in UTC.
+
+The schedule must be in one of 3 modes:
+- `MODE_ONCE` -- the mission is executed exactly once per day, at the indicted UTC time
+- `MODE_LOOP` -- the mission is executed a set number of times per day, starting at the indicated UTC time and repeating
+  at fixed intervals until the desired number of repetitions is reached
+- `MODE_TRANSIENT` -- the mission is run exactly once, and is not saved to the persistent storage. This is intended to
+  be used for deferred execution of a mission (e.g. `execute mission X in 5 minutes` would use a transient schedule)
+
+Deleting a mission from the Mission Manager will result in that mission being automatically removed from any schedules
+it was part of. If a schedule has no missions is is automatically disabled, and cannot be re-enabled until it contains
+at least one mission.
+
+### A Note on Schedule Timing
+
+Automatically-scheduled missions are executed at the provided time +/- 10 seconds.  The duration provided by the
+`next_schedule` topic and `get_next_schedule` service is therefore approximate. We recommend rounding the time to the
+nearest minute when displaying the countdown information to the user in any sort of GUI.
+
+## Topics Exported by Mission Scheduler
+
+### mission_scheduler/next_schedule
+
+**Message Type:** [cpr_mission_scheduler_msgs/msg/NextSchedule](definitions#scheduler-nextschedule)
+
+**Description:** Publishes the UUID and duration to the next scheduled mission.  If no missions are scheduled the UUID
+will be blank.
+
+**Update Rate:** 1Hz
+
+### mission_scheduler/state
+
+**Message Type:** [cpr_mission_scheduler_msgs/msg/StorageState](definitions#scheduler-state)
+
+**Description:** Publishes the current state of the database. Anything that needs to display the current set of
+schedules should subscribe to this topic
+
+**Update Rate:** Latched, publishes on state-change only
+
+## Services Exported by Mission Scheduler
+
+### mission_scheduler/add_mission
+
+**Service Type:** [cpr_mission_manager_msgs/srv/AddRemoveById](definitions#storage-addremovebyid)
+
+**Description:** Add a mission to an existing schedule.
+
+### mission_scheduler/clone_schedule
+
+**Service Type:** [cpr_mission_scheduler_msgs/srv/CloneSchedule](definitions#scheduler-cloneschedule)
+
+**Description:** Perform a deep-copy of an existing schedule
+
+### mission_scheduler/create_schedule
+
+**Service Type:** [cpr_mission_scheduler_msgs/srv/CreateSchedule](definitions#scheduler-createschedule)
+
+**Description:** Create a new schedule
+
+### mission_scheduler/delete_everything
+
+**Service Type:** [cpr_mission_manager_msgs/srv/DeleteEverything](definitions#storage-deleteeverything)
```

Comment:
Is the use of "Everything" consistent with the other parts of the code? If so, no problem. (My instinct would have been to call this "delete_all_schedules", but not if it means creating a new convention.)

Reply:
This was sort of a forward placeholder in case we had multiple data types. But for now that's not an issue so I'll rename it.